src/config/config.example.js

```diff
@@ -1,30 +1,28 @@
-import { ALREADY_LAUNCHED } from "@env";
+import {ALREADY_LAUNCHED} from '@env';
 import * as firebase from 'firebase';
 
 import '@firebase/auth';
 import '@firebase/firestore';
 import '@firebase/storage';
 
-
 const firebaseConfig = {
-    apiKey: "",
-    authDomain: "",
-    projectId: "",
-    storageBucket: "",
-    messagingSenderId: "",
-    appId: "",
-    measurementId: "Y"
-  };
+  apiKey: '',
+  authDomain: '',
+  projectId: '',
+  storageBucket: '',
+  messagingSenderId: '',
+  appId: '',
+  measurementId: 'Y',
```

Comment:
Just for curiosity, what this variable does?

Reply:
typo